src/site/markdown/maven-configuration.md

```diff
@@ -53,6 +53,7 @@ To modify this file, edit the template and regenerate.
 | `maven.logger.showThreadId` | `Boolean` | If you would like to output the current thread id, then set to true. Defaults to false. |  `false`  | 4.0.0 | User properties |
 | `maven.logger.showThreadName` | `Boolean` | Set to true if you want to output the current thread name. Defaults to true. |  `true`  | 4.0.0 | User properties |
 | `maven.logger.warnLevelString` | `String` | The string value output for the warn level. Defaults to WARN. |  `WARN`  | 4.0.0 | User properties |
+| `maven.maven3Personality` | `Boolean` | User property for controlling "maven personality". If set to {@code true} Maven will behave as previous major version, Maven 3. |  `false`  | 4.0.0 | User properties |
```

Comment:
```suggestion
| `maven.maven3Personality` | `Boolean` | User property for controlling "maven personality". If activated Maven will behave as previous major version, Maven 3. |  `false`  | 4.0.0 | User properties |
```
this is just an idea as true/false is kind of technical detail and quite known.
Might give business case more attention, rather then some data type used.